js/constants/messages.js

```diff
@@ -110,6 +110,9 @@ const messages = {
   RESPONSE_WINDOW_STATE: _,
   LAST_WINDOW_STATE: _,
   UNDO_CLOSED_WINDOW: _,
+  // Menu rebuilding
+  REQUEST_WINDOW_STATE_FOR_MENU: _,
+  RESPONSE_WINDOW_STATE_FOR_MENU: _,
```

Comment:
Just noticed we also have `UPDATE_APP_MENU` you should rename that to be more specific to be about bookmarks, and then maybe that is an indication that you should be more specific for perf reasons in the message names as I originally suggested in the first message in this thread.